data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessor.java

```diff
@@ -171,6 +184,17 @@ public Collection<Record<Event>> doExecute(final Collection<Record<Event>> recor
         return records;
     }
 
+    private String transformKey(String key) {
+        if(keyValueProcessorConfig.getTransformKey().equals(LOWERCASE_KEY)) {
+            key = key.toLowerCase();
+        } else if(keyValueProcessorConfig.getTransformKey().equals(UPPERCASE_KEY)) {
+            key = key.substring(0, 1).toUpperCase() + key.substring(1);
+        } else if(keyValueProcessorConfig.getTransformKey().equals(CAPITALIZE_KEY)) {
```

Comment:
nit: like above, I believe space after `else if` follows the conventions used by the project.